addons/mail/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,115 @@
+mail         / mail.message             / needaction_partner_ids (many2many): DEL relation: res.partner [renamed to notified_partner_ids]
+mail         / mail.message             / notified_partner_ids (many2many): NEW relation: res.partner [renamed from needaction_partner_ids]
+# Done in pre migration script
+
+mail         / mail.message             / res_id (integer)              : relation is now 'model' ('False') [nothing to do]
+mail         / mail.message             / res_id (integer)              : type is now 'many2one_reference' ('integer')
+# Nothing to do
+
+mail         / mail.message             / canned_response_ids (one2many): NEW relation: mail.shortcode [inverse for message_ids]
+mail         / mail.shortcode           / message_ids (many2one)        : NEW relation: mail.message
+# I think nothing to do
+# Commit https://github.com/odoo/odoo/commit/66a08369741369946df219ac92b8881711fbb484
+
+mail         / mail.notification        / email_status (selection)      : DEL selection_keys: ['bounce', 'canceled', 'exception', 'ready', 'sent'] [renamed to email_status]
+mail         / mail.notification        / notification_status (selection): NEW selection_keys: ['bounce', 'canceled', 'exception', 'ready', 'sent'], hasdefault [renamed from email_status]
+# Done in pre-migration script
+
+mail         / mail.notification        / notification_type (selection) : NEW required, selection_keys: ['email', 'inbox'], req_default: function, hasdefault
+# Done in post-migration script. Tips from commit: [notification_type: notification.is_email and 'email' else 'inbox']
+
+mail         / mail.notification        / read_date (datetime)          : NEW
+# I think nothing to do
+# Commit https://github.com/odoo/odoo/commit/0c37b9fb39005eab70ee8b39da84f79c981b2695
+
+mail         / mail.notification        / is_email (boolean)            : DEL
+# Nothing to do
+
+mail         / mail.tracking.value      / track_sequence (integer)      : DEL [renamed to tracking_sequence]
+mail         / mail.tracking.value      / tracking_sequence (integer)   : NEW hasdefault [renamed from track_sequence]
+# Done in pre-migration script
+
+mail         / res.partner              / email_normalized (char)       : NEW isfunction: function, stored
+# Check that Odoo core support this change
+
+mail         / res.users                / out_of_office_message (char)  : NEW
+# I think nothing to do
+# Commit https://github.com/odoo/odoo/commit/b60224c41cb6c9817fd328bb60b21681e619fb26
+
+---XML records in module 'mail'---
+NEW ir.actions.act_window: mail.mail_activity_action
+NEW ir.cron: mail.ir_cron_delete_notification (noupdate)
+NEW ir.ui.menu: mail.menu_mail_activities
+NEW ir.ui.view: mail.res_partner_view_activity
+NEW mail.activity.type: mail.mail_activity_data_upload_document (noupdate)
+# Nothing to do
+
+DEL ir.ui.view: mail.qunit_mobile_suite
+# Done in post-migration. Remove by xmlid
```

Comment:
This record is not noupdate. I believe the delete_records_safely_by_xml_id only needs to be called on noupdate=1 records that have been removed from the module data, because OpenUpgrade, like Odoo, will clean up other records automatically.